api/src/pcapi/core/finance/utils.py

```diff
@@ -72,6 +72,12 @@ def install_template_filters(app: Flask) -> None:
     app.jinja_env.filters["fr_currency_opposite"] = fr_currency_opposite_filter
 
 
+def format_currency_for_backoffice(amount: float | decimal.Decimal, use_xpf: bool = False) -> str:
+    if use_xpf:
+        return f"{euros_to_xpf(amount)}".replace(",", "\u202f")
+    return f"{amount:,.2f}".replace(",", "\u202f").replace(".", ",")
```

Comment:
Dès lors qu'on renvoie une chaîne je serais plutôt partisan d'inclure directement la devise ici, et ne plus avoir besoin de l'ajouter dans les fonctions utilisatrices.
```suggestion
        return f"{euros_to_xpf(amount)} €".replace(",", "\u202f")
    return f"{amount:,.2f} CFP".replace(",", "\u202f").replace(".", ",")
```